Client/game_sa/CBikeSA.h

```diff
@@ -14,6 +14,25 @@
 #include <game/CBike.h>
 #include "CVehicleSA.h"
 
+namespace eBikeNodes
+{
+    enum
+    {
+        NONE = 0,
+        CHASIS,
```

Comment:
Should be CHASSIS. The same for below one.

Reply:
Done, I did a little more cleaning